Assets/Joystick Pack/Scripts/Editor/FloatingJoystickEditor.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEditor;
+
+[CustomEditor(typeof(FloatingJoystick))]
+public class FloatingJoystickEditor : JoystickEditor
+{
+    public override void OnInspectorGUI()
+    {
+        base.OnInspectorGUI();
+
+        if (background != null)
+        {
+            RectTransform backgroundRect = (RectTransform)background.objectReferenceValue;
+            backgroundRect.anchorMax = Vector2.zero;
+            backgroundRect.anchorMin = Vector2.zero;
+            backgroundRect.pivot = center;
+        }
+    }
+}
```

Comment:
no newline at the end of the file @ Assets/Joystick Pack/Scripts/Editor/FloatingJoystickEditor.cs